plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -407,7 +457,7 @@ def run_inspection(file_filter, options, do_extract, file_store_tmp, file_store_
       dirs_todo.push(*managed)
 
       # unmanaged files are simply stored
-      managed, unmanaged = files.keys.partition{ |d| rpm_files.has_key?(find_dir + d) }
+      managed, unmanaged = files.keys.partition{ |d| inspected_files.has_key?(find_dir + d) }
```

Comment:
`Hash#has_key?` is deprecated in favor of `Hash#key?`.<br>Space missing to the left of {.